tests/unit/wingsail/common/test_lut.py

```diff
@@ -0,0 +1,63 @@
+import math
+
+import numpy as np
+import pytest
+from scipy import interpolate
+
+from controller.common.lut import LUT
+
+
+class TestLUT:
+    # Intialize lookup table
+    look_up_table = [[50000, 5.75], [100000, 6.75], [200000, 7], [500000, 9.75], [1000000, 10]]
+
+    def test_LUT_constructor(self):
+        # set up
+        testLUT = LUT(self.look_up_table)
+
+        # test that LUT return a known value
+        assert math.isclose(testLUT(50000), 5.75)
+
+    def test_unknown_interpolation_exception(self):
+        with pytest.raises(ValueError):
+            testLUT = LUT(self.look_up_table, "gabagool")
+
+    def test_invalid_table_exception(self):
```

Comment:
I recommend passing an invalid numpy array as well.

Reply:
added to test:

```

    @pytest.mark.parametrize(
        "invalid_table",
        [
            np.array([[10000, 10000, 10000], [1, 1, 1]]),
            np.array([10000, 10000, 10000]),
            np.array([[[0, 1]], [[0, 1]], [[0, 1]]]),
        ],
    )
    def test_invalid_numpy_array_exception(self, invalid_table):
        with pytest.raises(ValueError):
            testLUT = LUT(invalid_table)
```